pmoves/services/pmoves-yt/yt.py

```
@@ -54,7 +55,64 @@ def clear_cache() -> None:
+                except Exception:
+                    interval = 86400
+                async def _periodic_docs_sync():
+                    while True:
+                        await asyncio.sleep(interval)
+                        try:
+                            docs = collect_yt_dlp_docs()
+                            sync_to_supabase(docs)
+                            logger.info("yt-dlp docs synced (periodic)")
+                        except Exception as exc:
+                            logger.warning("periodic docs sync failed: %s", exc)
+                asyncio.create_task(_periodic_docs_sync(), name="pmoves-yt-docs-sync")
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)